packages/sdk-js/src/GrowthBook.ts

```diff
@@ -1278,6 +1353,40 @@ export class GrowthBook<
     return false;
   }
 
+  public getRedirectUrl(): string {
+    return this._redirectedUrl;
+  }
+
+  private _getNavigateFunction():
+    | null
+    | ((url: string) => void | Promise<void>) {
+    if (this._ctx.navigate) {
+      return this._ctx.navigate;
+    } else if (isBrowser) {
+      return (url: string) => {
+        window.location.href = url;
+      };
+    }
+    return null;
+  }
+
+  private _setAntiFlicker() {
```

Comment:
antiFlicker might actually introduce more flicker. i.e. Growthbook takes a while to load, loads-> setAntiFlicker so user is actually getting more flicker

Reply:
We decided this is okay as long as `antiFlicker` remains opt in